lib/SiftClient.php

```diff
@@ -310,6 +310,172 @@ public function getOrderDecisions($order_id, $opts = array()) {
 
+    /**
+     * Gets a list of configured decisions.
+     *
+     * @param array $opts  Array of optional parameters for this request:
+     *     - string 'account_id': by default, this client's account ID is used.
+     *     - int 'timeout': By default, this client's timeout is used.
+     *     - array 'abuse_types': filters decisions which can be apply to
+     *       listed abuse types
+     *     - string 'entity_type': filters on decisions which can be applied to
+     *       a specified entity_type
+     *     - string 'next_ref': url that will fetch the next page of decisions
+     *     - int 'limit': sets the max number of decisions returned
+     *     - int 'from': will return the next decision from the index given up
+     *       to the limit.
+     *
+     */
+    public function getDecisions($opts = array()) {
+      $this->mergeArguments($opts, array(
+        'account_id' => $this->account_id,
+        'timeout' => $this->timeout,
+        'abuse_types' => null,
+        'entity_type' => null,
+        'next_ref' => null,
+        'limit' => null,
+        'from' => null
+      ));
+
+      $params = array();
+
+      if ($opts['next_ref']) {
+        $url = $opts['next_ref'];
+      } else {
+        $url = (self::API3_ENDPOINT . '/v3/accounts/' . $opts['account_id']
+                . '/decisions');
+
+        if ($opts['abuse_types']) $params['abuse_types'] = implode(',', $opts['abuse_types']);
+        if ($opts['entity_type']) $params['entity_type'] = $opts['entity_type'];
+        if ($opts['limit']) $params['limit'] = $opts['limit'];
+        if ($opts['from']) $params['from'] = $opts['from'];
+      }
+
+      try {
+        $request = new SiftRequest($url, SiftRequest::GET, $opts['timeout'],
+          self::API3_VERSION,
+          array('auth' => $this->api_key . ':'));
+
+        return $request->send();
+      } catch (Exception $e) {
+        return null;
+      }
+    }
+
+    /**
+     * Apply a decision to a user. Builds url to apply decision to user and
+     * delegates to applyDecision
+     *
+     * @param array $opts  Array of optional parameters for this request:
+     *     - string 'account_id': by default, this client's account ID is used.
+     *     - int 'timeout': By default, this client's timeout is used.
+     *     - string 'decision_id': The decision that will be applied to a user
+     *     - string 'source': the source of the decision, i.e. MANUAL_REVIEW,
+     *     AUTOMATED_RULE, CHARGEBACK
+     *     - string 'analyst': when the source is MANUAL_REVIEW, an analyst
+     *     identifier must be passed.
+     *     - string 'user_id': the id of the user that will get this decision
+     *     - string 'description': free form text adding context to why this
+     *     decision is being applied.
+     *     - int 'time': Timestamp of when a decision was applied, mainly used
+     *     for backfilling
+     *
+     */
+    public function applyDecisionToUser($opts = array()) {
```

Comment:
Required parameters -- i.e., `user_id` and `decision_id` -- should be included as top-level parameters to this method, rather than in the `$opts` array.